docs/reference/feast-cli-commands.md

```diff
@@ -205,12 +211,18 @@ types:
 - FEATURE_VIEW
 withSubclasses: true
 namePattern: transformed_conv_rate
+requiredTags:
+  key1: value1
+  key2: value2
 actions:
 - READ
 policy:
   roleBasedPolicy:
     roles:
     - reader
+tags:
+  key1: value1
```

Comment:
could you please differentiate key names and values b/w `required_tags` and `tags`?